src/mutation.ts

```diff
@@ -329,7 +373,8 @@ export class Mutation {
    * @returns {object}
    * @private
    */
-  toProto() {
+  toProto(): JsonObj {
+    // tslint:disable-next-line no-any
```

Comment:
I think this should be [`IMutateRowRequest`](https://github.com/googleapis/nodejs-bigtable/blob/master/proto/bigtable.d.ts#L709), should be able to drop the `no-any` comment too.